FirstClasses/multiples/src/multiples.erl

```diff
@@ -1,5 +1,39 @@
 -module(multiples).
 
 -export([thousand_sum/0]).
+-export([generate_input_collection/1]).
+-export([filter_collection/1]).
 
-thousand_sum() -> 1.
+thousand_sum() -> 
+    FilterCollection = filter_collection(generate_input_collection(1000)),
+    lists:sum(FilterCollection).
+
+generate_input_collection(Lenght) ->
+    case Lenght of
+        0  ->
+           [];
+        L when L - 1 == 0 ->
```

Comment:
I am not 100 % sure what is this case for, as `lists:seq/2` works even if starting and ending elements are equal. Try it in a console:
```erlang
1> lists:seq(1,1).
[1]
```
Moreover `L when L - 1 == 0 ->` can be simplified to `L when L == 1 ->` which can be simplified (even more) to `1 ->` :)

Reply:
hm... i dont remeber why I created this case. It looks like it could be [1] because I didnt know that  lists:seq(1,1) wold works. Thank you!